releases/v21.1.0-alpha.2.md

```diff
@@ -0,0 +1,544 @@
+- The `Timestamp` field of structured notable events is now numeric, and encodes a number of nanoseconds since the unix epoch. [#58070][#58070] {% comment %}doc{% endcomment %}
+- The Health API now checks that the SQL server is ready to accept clients when a readiness check is requested. [#59350][#59350] {% comment %}doc{% endcomment %}
+
+### DB Console changes
+
+- Minor style changes to represent new branding palette. [#57130][#57130]
+- Changed default per-page value on [Transactions page](../v21.1/ui-transactions-page.html) to 20; minor style updates. [#57824][#57824]
+- Added a timeseries graph indicating statement denials due to feature flags on the [SQL Dashboard](../v21.1/ui-sql-dashboard.html) of DB Console. [#57533][#57533] {% comment %}doc{% endcomment %}
+- Updated labels on [Hardware Dashboard](../v21.1/ui-hardware-dashboard.html) to be more accurate. [#57224][#57224]
+- Link back to [Statements page](../v21.1/ui-statements-page.html) from [Statement Details page](../v21.1/ui-statements-page.html#statement-details-page) will always link to the Statements list instead of invoking "back" action on the browser. [#57975][#57975]
+- On the [Sessions page](../v21.1/ui-sessions-page.html), any `age` label has been replaced with `duration` and `txn` has been replaced with `transaction`. The actual metrics remain unchanged. [#58616][#58616] {% comment %}doc{% endcomment %}
+- Renamed the `CPUs` metric column to `vCPUs` in the Node List on the [Cluster Overview page](../v21.1/ui-cluster-overview-page.html). [#58495][#58495] {% comment %}doc{% endcomment %}
+- Added Open SQL Transactions to [SQL Dashboard](../v21.1/ui-sql-dashboard.html) and renamed `SQL Queries` to `SQL Statements`. Removed references to "Distributed SQL Queries" when metrics really are discussing all SQL queries. [#57477][#57477] {% comment %}doc{% endcomment %}
+
+### Bug fixes
+
+- Fixed a bug in [`RESTORE`](../v21.1/restore.html) where some unusual range boundaries in interleaved tables caused an error. [#58219][#58219]
+- Previously, CockroachDB would encounter an internal error when performing `JSONB - String` operation via the [vectorized execution engine](../v21.1/vectorized-execution.html). This has been fixed. The bug was introduced in v20.2.0. [#57349][#57349]
+- In rare situations, an automated replication change could result in a loss of quorum. This would require down nodes and a simultaneous change in the replication factor. Note that a change in the replication factor can occur automatically if the cluster is comprised of fewer than five available nodes. Experimentally the likelihood of encountering this issue, even under contrived conditions, was small. [#56735][#56735]
```

Comment:
> This would require down nodes and a simultaneous change in the replication factor.

I'm not sure what the intent is, but I believe this needs an edit.